products/protractor.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Protractor
+category: framework
+tags: google javascript-runtime test
+iconSlug: protractor
+permalink: /protractor
+versionCommand: npm list protractor
+
+releaseDateColumn: true
+activeSupportColumn: Community Support
+extendedSupportColumn: Commercial Support
+eolColumn: false
+
+auto:
+-   npm: protractor
+
+identifiers:
+-   purl: pkg:npm/protractor
+
+releases:
+-   releaseCycle: "7.0"
+    support: 2023-08-31
+    eol: true
+    extendedSupport: true
+    latest: "7.0.0"
+    latestReleaseDate: 2020-05-13
+    releaseDate: 2020-05-13
+    link: https://github.com/angular/protractor/blob/release-7.0/CHANGELOG.md#700
+
+-   releaseCycle: "6.0"
+    support: 2023-08-31
+    eol: true
+    extendedSupport: false
+    latest: "6.0.0"
+    latestReleaseDate: 2019-03-23
+    releaseDate: 2019-03-23
+    link: https://github.com/angular/protractor/blob/master/CHANGELOG.md#600
+
+-   releaseCycle: "5.0"
+    support: 2023-08-31
+    eol: true
+    extendedSupport: false
+    latest: "5.4.4"
+    latestReleaseDate: 2020-04-16
+    releaseDate: 2017-01-09
+    link: https://github.com/angular/protractor/blob/release-5.4/CHANGELOG.md#544
+
+-   releaseCycle: "4.0"
+    support: 2023-08-31
+    eol: true
+    extendedSupport: false
+    latest: "4.0.14"
+    latestReleaseDate: 2016-12-20
+    releaseDate: 2016-06-12
+    link: https://github.com/angular/protractor/blob/master/CHANGELOG.md#4014
+
+-   releaseCycle: "3.0"
+    support: 2023-08-31
+    eol: true
+    extendedSupport: false
+    latest: "3.3.0"
+    latestReleaseDate: 2016-04-25
+    releaseDate: 2015-11-17
+    link: https://github.com/angular/protractor/blob/master/CHANGELOG.md#330
+
+-   releaseCycle: "2.0"
+    support: 2023-08-31
+    eol: true
+    extendedSupport: false
+    latest: "2.5.1"
+    latestReleaseDate: 2015-10-08
+    releaseDate: 2015-03-18
+    link: https://github.com/angular/protractor/blob/master/CHANGELOG.md#251
+
+-   releaseCycle: "1.0"
+    support: 2023-08-31
+    eol: true
+    extendedSupport: false
+    latest: "1.8.0"
+    latestReleaseDate: 2015-03-02
+    releaseDate: 2014-07-21
+    link: https://github.com/angular/protractor/blob/master/CHANGELOG.md#180
+
+-   releaseCycle: "0.1.0"
+    support: 2023-08-31
+    eol: true
+    extendedSupport: false
+    latest: "0.24.2"
+    latestReleaseDate: 2014-06-17
+    releaseDate: 2013-02-23
+    link: https://github.com/angular/protractor/blob/master/CHANGELOG.md#0242
```

Comment:
```suggestion

```

We usually don't document versions < 1.0.

Reply:
Got it, removed.